pom.xml

```diff
@@ -808,6 +808,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
Please relocate the driver and its dependencies into our package in the maven shade plugin configuration.

Reply:
Done :)